chandra_aca/tests/test_maude_decom.py

```diff
@@ -319,6 +319,8 @@ def test_vcdu_vs_level0():
             os.path.join(
                 os.path.dirname(__file__), "data", f"test_level0_{slot}.fits.gz"
             ),
+            # Undocumented way to stop UnitsWarning 'DN' did not parse warning. See
+            # https://github.com/astropy/astropy/issues/14330#issuecomment-1406538512
```

Comment:
I was wondering if this belonged in some code and not just the test, but I think the mica L0 reading still uses pytables directly.

Reply:
Discussed. This should be fixed upstream in astropy and there is an issue but it is stalled.